Libraries/LibGUI/GColumnsView.cpp

```diff
@@ -136,6 +136,19 @@ void GColumnsView::paint_event(GPaintEvent& event)
     }
 }
 
+void GColumnsView::select_all()
+{
+    selection().clear();
+    for (int column_index = 0; column_index < m_columns.size(); ++column_index) {
```

Comment:
Don't iterate over column indexes, iterate over columns themselves

Reply:
That being said I'm not sure if you even *should* iterate over columns — should `GColumnsView` select all the displayed hierarchy, or only items on the "current" level?